src/services/database/__tests__/registration-request-repository.integration.test.js

```diff
@@ -1,9 +1,12 @@
+import { v4 as uuid } from 'uuid';
 import ModelFactory from '../../../models';
 import { modelName, Status } from '../../../models/registration-request';
 import {
+  getNhsNumberByConversationId,
   getRegistrationRequestStatusByConversationId,
   updateRegistrationRequestStatus
-} from '../../database/registration-request-repository';
+} from '../registration-request-repository';
+import {NhsNumberNotFoundError} from "../../../errors/errors";
```

Comment:
Incredibly small request for consistency! Totally optional 😄 

```suggestion
import { NhsNumberNotFoundError } from "../../../errors/errors";
```